main.py

```diff
@@ -44,7 +44,12 @@ async def websocket_endpoint(websocket: WebSocket):
         except WebSocketDisconnect:
             print("Client disconnected.")
             break
-    await websocket.close()
+        # Since we're already handling the specific exception above and logging the exception
+        # we can suppress pylint W0703 - Catching too general exception Exception (broad-except)
+        except Exception as error: # pylint: disable=W0703
+            print(f"ERROR: {error}")
```

Comment:
```suggestion
            print(f"Error: {error}")
```
We can also make it red-colored using `console.py`.

Reply:
Should I change the color of the other print? (the one for the client disconnect)

Reply:
Since it's expected, I'd think of it as an info message instead of an error.

Reply:
Added. Had to change the printing in console.py to use f-strings for pylint